openassessment/assessment/api/peer.py

```diff
@@ -168,7 +179,7 @@ def get_score(submission_uuid, peer_requirements):
         assessment__score_type=PEER_TYPE
     ).order_by('-assessment')
 
-    submission_finished = items.count() >= peer_requirements["must_be_graded_by"]
+    submission_finished = items.count() >= _required_peer_grades(workflow, peer_requirements)
```

Comment:
I think it might make sense to move this entire evaluation into the function, i.e. call the function: `_check_submission_finished` or `_is_submission_finished` etc..